packages/morph/src/page/Editor/Workspace.tsx

```diff
@@ -27,13 +27,32 @@ This is **bold** text.
         EditorView.lineWrapping,
         markdown(),
         inlineMarkdownExtension,
-        vim()
-      ]
+        vim(),
+        EditorView.theme({
+          "&": {
+            maxWidth: "75rem",
+            margin: "0 auto",
+            padding: "2rem 4rem",
+          },
+          ".cm-gutters": {
+            display: "none",
+          },
+          ".cm-scroller": {
+            overflowX: "hidden",
+          },
+          ".cm-activeLine": {
+            backgroundColor: "transparent",
+          },
+          "&.cm-editor.cm-focused": {
+            outline: "none",
+          },
+        }),
+      ],
```

Comment:
can we strictly do this in either JSX or scss?

let's try to prevent these mix pattern imo